storage/invoke_test.go

```diff
@@ -33,32 +33,97 @@ func TestInvoke(t *testing.T) {
 	// returns with the right error.
 
 	for _, test := range []struct {
-		count      int   // Number of times to return retryable error.
-		initialErr error // Error to return initially.
-		finalErr   error // Error to return after count returns of retryCode.
+		count             int   // Number of times to return retryable error.
+		initialErr        error // Error to return initially.
+		finalErr          error // Error to return after count returns of retryCode.
+		retry             *retryConfig
```

Comment:
I see that you don't actually use `retry` in any of the test cases, are you planning on adding this later?

Reply:
Actually I think I'll just merge this PR as is since you are out-- I can make a separate one adding retry test cases.